src/test/scala/uk/gov/hmrc/auth/Await.scala

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmrc.auth
+
+import scala.concurrent.{Await, Future}
+
+trait Await {
```

Comment:
I know, I didn't want to add a dependency just for that and otherwise we don't need hmrc test.